### ISSUE TITLE
Final exam schedule

### ISSUE THREAD

**Issue by iqfareez** [iqfareez/iium_schedule#66]
Add ability to add final exam schedule

**Comment by iqfareez** [comment]
In this [pre-release](https://github.com/iqfareez/iium_schedule/releases/tag/1.2.0-pre.1%2B27). A POC of final exam schedule has been added.

The final exam will look to the saved schedule, and try to fetch the schedule information from the given course code. So, this way, no manual input from the user is required.  However, the [source](https://www.iium.edu.my/division/amad/academic-calendarimportant-dates) didn't provide much information needed such as exam venue, seats etc. In addition, adding the final exam data to the [albiruni-api](https://github.com/iqfareez/albiruni-api) is complicated. This includes downloading PDF from source and convert it into Excel file to do further processing.

So, I'm thinking to go with importing from I-ma'luum (yes, just like how we import course schedule). The plus side is we got all the information such as venue, time etc. :+1:

![i-ma'luum final exam](https://user-images.githubusercontent.com/60868965/232027501-dc95d678-ffdb-4f0d-95ad-a8fbbb7d7129.png)
